opentelemetry-otlp/src/span.rs

```diff
@@ -70,6 +82,18 @@ pub enum TraceExporter {
         /// The Grpc trace exporter
         trace_exporter: GrpcioTraceServiceClient,
     },
+    #[cfg(feature = "http-proto")]
+    /// Trace Exporter using HTTP transport
+    HTTP {
```

Comment:
```suggestion
    Http {
```

I think [the naming convention](https://rust-lang.github.io/api-guidelines/naming.html#casing-conforms-to-rfc-430-c-case) requires camel case for enum variants

Reply:
I'll change it, thanks.